src/DTO/DefaultStrategy.php

```diff
@@ -47,6 +49,14 @@ public function getSegments(): array
         return $this->segments;
     }
 
+    /**
+     * @return array<Variant>
+     */
+    public function getVariants(): array
+    {
+        return $this->variants;
+    }
```

Comment:
This is wrong, it's not the strategy that contains the variants, it's a feature that contains them.

Reply:
This is the new feature https://docs.getunleash.io/reference/strategy-variants. Now strategies also contain variants.